modules/core/src/main/scala/org/scalasteward/core/data/ProcessResult.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2018-2019 Scala Steward contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.scalasteward.core.data
+
+sealed trait ProcessResult extends Product with Serializable
+
+case object Ignored extends ProcessResult
+case object Updated extends ProcessResult
```

Comment:
Minor nitpick: Could you put these two objects into a new `ProcessResult` companion? I prefer if these names do not pollute the package namespace.